test/unit.h

```diff
@@ -70,24 +79,16 @@ extern "C" {
 		return check_plan();	// print resume
 	}
 @endcode
-
-
 */
 
 /* private function, use ok(...) instead */
 void
 _ok(int condition, const char *expr, const char *file, int line,
     const char *fmt, ...);
 
-/* private function, use note(...) or diag(...) instead */
+/* private function */
 void _space(FILE *stream);
 
-#define msg(stream, ...) ({ _space(stream); fprintf(stream, "# ");            \
-	fprintf(stream, __VA_ARGS__); fprintf(stream, "\n"); })
-
-#define note(...) msg(stdout, __VA_ARGS__)
-#define diag(...) msg(stderr, __VA_ARGS__)
-
```

Comment:
We have these functions in the main Tarantool repository. Let's leave them here as well.

Reply:
Done.